ts/Core/Templating.ts

```diff
@@ -289,7 +289,7 @@ function format(str = '', ctx: any, chart?: Chart): string {
     }
 
     // Execute
-    matches.forEach((match): void => {
+    matches.forEach(function (match): void {
```

Comment:
Was this change necessary or just a remaining debugging step? I can't see that we use the function scope in the solution?

Reply:
It's a remain from debugging step, but I left it with `function` word, because I had an issue with wrong `chart` instance, now it works correctly with the arrow function  🤷‍♂️  I reverted it.